### PR TITLE
Spectrafit tests p

### PR DESCRIPTION
**Pull request by ParkerSteichen**
Added one more test function and updated both spectrafit and test_spectrafit to PEP8 (as good as we want)

### CODE REVIEW

File: ramandecompy/tests/test_spectrafit.py [hunk 1]
```diff
@@ -192,80 +199,93 @@ def test_build_custom_model():
     """
     docstring
```
Comment (erasmuss) [inline ramandecompy/tests/test_spectrafit.py]: need doc string here

File: ramandecompy/tests/test_spectrafit.py [hunk 2]
```diff
@@ -192,80 +199,93 @@ def test_build_custom_model():
...
     try:
         spectrafit.build_custom_model(X_TEST, Y_TEST, peaks, peaks_add, plot_fits=1)
     except TypeError:
-            print('An int was passed to the function, and was handled well with a TypeError.')
+        print('An int was passed to the function, and was handled well with a TypeError.')
 
 
 def test_apply_old_model():
     """
     docstring
```
Comment (erasmuss) [inline ramandecompy/tests/test_spectrafit.py]: need doc string here

not sure what this function is...

File: ramandecompy/tests/test_spectrafit.py [hunk 3]
```diff
@@ -275,7 +295,7 @@ def test_superimpose_next():
     # initialize inputs
```
Comment (erasmuss) [inline ramandecompy/tests/test_spectrafit.py]: need doc string

File: ramandecompy/tests/test_spectrafit.py [hunk 4]
```diff
@@ -299,3 +319,31 @@ def test_superimpose_next():
     os.remove(hdf5_filename)
 
 
+def test_superimpose_set():
+    """
+    docstring
```
Comment (erasmuss) [inline ramandecompy/tests/test_spectrafit.py]: need doc string

**Comment by erasmuss** [comment]
@ParkerSteichen reviewed and need doc strings, not sure what one of the functions is doing/is used for - made comment at that location. please update before merge is approved, thanks!

File: ramandecompy/tests/test_spectrafit.py (hunk 2 above)
Comment (ParkerSteichen) [reply ramandecompy/tests/test_spectrafit.py]: This function is used within the `superimpose_next()` function which is explained in the docstrings in spectrafit itself.

**Comment by ParkerSteichen** [comment]
After adding requested changes I will merge now with the master and delete the branch

**Comment by ParkerSteichen** [comment]
I restored this branch because I actually realized that I was one unit test short!! @erasmuss I will update tonight and you can review again

**Pull request merged**